merge request.md

```diff
@@ -0,0 +1,68 @@
+1. template
```

Comment:
This file needs to be removed from the repository.
Because it doesn't belong to this project.